### PR TITLE
Template updates to package.yamls

### PR DESCRIPTION
**Pull request by kariharju**
Changed documentation to point to relative file.
Added version -field back

### CODE REVIEW

File: templates/advanced/package.yaml [hunk 1]
```diff
@@ -4,15 +4,18 @@ name: Advanced Template Package
 # Required: A description of what's in the action package.
 description: Action package description
 
-# Required: A link to where the documentation on the package lives.
-documentation: https://github.com/...
+# Required: Point to main documentation markdown file for the package.
+documentation: README.md
```
Comment (cmin764) [inline templates/advanced/package.yaml]: Have we decided on picking configuration over convention? Like, what would be an use-case where the `documentation:` field won't be a _README.md_? Normally, any action pack contains the main _README.md_, which can also point to additional documentation found in the _docs_ dir if needed.

The _package.png_ idiom works great so far, so what would be an advantage here on bringing them in the _package.yaml_?

File: templates/advanced/package.yaml [hunk 2]
```diff
@@ -4,15 +4,18 @@ name: Advanced Template Package
...
+
+# Package version number, recommend using semver.org
+version: 0.0.1
 
 dependencies:
   conda-forge:
   - python=3.10.14
   - uv=0.2.4
```
Comment (cmin764) [inline templates/advanced/package.yaml:13]: I saw some variations of `uv=0.2.6` here, while the latest is `uv=0.2.9`. What's the rationale when picking the right version and why would **0.2.4** be better than the latest one which is **0.2.9**?

**Review by cmin764 — COMMENTED** [review]
Placed two questions that should be answered clearly before on proceeding updating those, so we'd learn on how to act about it from now on.

File: templates/advanced/package.yaml (hunk 1 above)
Comment (kariharju) [reply templates/advanced/package.yaml]: Good point, we agreed to convention on this.

File: templates/advanced/package.yaml (hunk 2 above)
Comment (kariharju) [reply templates/advanced/package.yaml:13]: Bi-weekly update cadence needs to be set.. so not everyone updating to the latest "whenever" but a set time when we update and test all to a specific uv version.
We cannot have 12 different environment contents, we rather need more of a "golden env." that all of these try to use.

**Review by el-mike — APPROVED** [review]

**Pull request merged**